src/overmap_ui.cpp

```diff
@@ -616,7 +616,8 @@ void draw( const catacurses::window &w, const catacurses::window &wbar, const tr
     if( !note_text.empty() ) {
         const size_t pos = std::get<2>( get_note_display_info( note_text ) );
```

Comment:
If you add a `const std::tuple<char, nc_color, size_t> note_info = get_note_display_info( note_text );` above `pos` you can calculate the info once, making affected lines:
`const size_t pos = std::get<2>( note_info )` and `corner_text.emplace_back( std::get<1>( note_info ), note_text.substr( pos ) );`

Reply:
Hmm that would be more efficient so that the info is only grabbed once and only one object is created. Ill add that and commit :)